{{cookiecutter.project_slug}}/Makefile

```diff
@@ -7,7 +7,18 @@ SHELL=/bin/bash
 # ---------------------------------
 #
 # Add any targets specific to the current project in here.
-
+{%- if cookiecutter.multilingual == 'y'  %}
+translations: ## Create/compile dummy translations
+	rm -rf locale/{en,uni}
+	mkdir -p locale/{en,uni}/LC_MESSAGES
+	./manage.py makemessages --verbosity=0
+
+	podebug --rewrite=unicode locale/{en,uni}/LC_MESSAGES/django.po
+	sed -i -e 's/.Plural-Forms:.*//' locale/en/LC_MESSAGES/django.po
+	sed -i -e 's/.Plural-Forms:.*//' locale/uni/LC_MESSAGES/django.po
+	find locale -name '*-e' -delete
+	cd locale && ../manage.py compilemessages --verbosity=0
```

Comment:
Do we want `cd locale` in here? I was getting this error `CommandError: This script should be run from the Django Git checkout or your project or app tree, or with the settings module specified.` when I ran `make translations`, but taking `cd locale` out resolved it.